app/views/brexit_checker/results.html.erb

```diff
@@ -67,16 +67,12 @@
       </div>
     </div>
 
-    <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible govuk-section-break--bold">
-    <div class="govuk-grid-row govuk-!-margin-bottom-6">
-      <div class="govuk-grid-column-one-third">
-        <%= render 'results_answers', answers: format_criteria_list(@criteria) %>
-      </div>
-      <div class="govuk-grid-column-two-thirds">
-        <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible govuk-section-break--bold govuk-section-break--visible-mobile-only">
-        <%= render 'action_audiences', audiences: format_action_audiences(@actions), action_based_description: action_based_description %>
-      </div>
-    </div>
+    <% if @business_results.any? || @citizen_results.any? %>
```

Comment:
Could change this a bit and just put 
'''if @business_results.empty? && @citizens_results.empty?" on the last render